luminance-glutin/Cargo.toml

```diff
@@ -17,7 +17,7 @@ maintenance = { status = "actively-developed" }
 
 [dependencies]
 gl = "0.13"
-glutin = { version = "0.21", default-features = false }
+glutin = { version = "0.22.0-alpha5", default-features = false }
```

Comment:
Unfortunately, I cannot accept that line now as it would make **luminance-glutin** depend on a beta version, which is not something that I want to authorize.